docs/openapi/openapi.yaml

```diff
@@ -265,7 +265,7 @@ components:
             - CERTIFICATE_PROVIDER_SIGN
             - ATTORNEY_SIGN
             - TRUST_CORPORATION_SIGN
-            - PERFECT
+            - STATUTORY_WAITING_PERIOD
```

Comment:
I think if we change this it will break MRLPA because [they use it in fixtures](https://github.com/ministryofjustice/opg-modernising-lpa/blob/97d05f35d57ca2ef2647cb1a14889375599e4efe/internal/lpastore/update.go#L64-L70).

We can probably get away with co-releasing changes to both repos, but should at least line up the change in opg-modernising-lpa first (if MRLPA devs are cool with that?)

Reply:
Yeah, we [pull](https://github.com/ministryofjustice/opg-modernising-lpa/blob/main/docker/mock-lpa-store/mock-lpa-store-config.yaml#L2) this spec in as the basis for our mock lpa store so it needs to be valid.

To uncouple this we could add `STATUTORY_WAITING_PERIOD` and leave `PERFECT` until changes are made in MRLPA? I can ticket the change up with this update in an AC which should (hopefully) mean this doesn't get forgotten about if we're happy with that approach?